packages/component/src/Transcript/LiveRegionTranscript.tsx

```diff
@@ -1,47 +1,28 @@
 import { hooks } from 'botframework-webchat-api';
-import classNames from 'classnames';
-import PropTypes from 'prop-types';
-import React, { useEffect, useMemo, useRef } from 'react';
-import type { FC, RefObject, VFC } from 'react';
 import type { WebChatActivity } from 'botframework-webchat-core';
+import type { RefObject } from 'react';
+import React, { memo, useEffect, useMemo, useRef } from 'react';
 
-import isPresentational from './LiveRegion/isPresentational';
 import LiveRegionActivity from '../LiveRegion/LiveRegionActivity';
-import LiveRegionSendFailed from './LiveRegion/SendFailed';
-import LiveRegionTwinComposer from '../providers/LiveRegionTwin/LiveRegionTwinComposer';
 import tabbableElements from '../Utils/tabbableElements';
 import useLocalizeAccessKey from '../hooks/internal/useLocalizeAccessKey';
-import useQueueStaticElement from '../providers/LiveRegionTwin/useQueueStaticElement';
-import useStyleToEmotionObject from '../hooks/internal/useStyleToEmotionObject';
 import useSuggestedActionsAccessKey from '../hooks/internal/useSuggestedActionsAccessKey';
+import { useQueueStaticElement } from '../providers/LiveRegionTwin';
```

Comment:
I would love to see the newer `usePushToLiveRegion()` (or as I renamed to `useLiveRegion()`) to be able to superseded `useQueueStaticElement()` in all cases.

The only exception is the usage in this component.

`<LiveRegionTranscript>` need a render loop and inspect the DOM element to see if certain live region text need to be added based on the rendered content.

To use the new `useLiveRegion()` hook in this component, we will need to introduce another render loop, then it would call the `useLiveRegion`.

It seems the new `useLiveRegion()` is not very capable on this scenario. Makes me think if the new approach is right or not.

When we want to push something to live region, it could be at anytime, inside or outside of render loop. In this component, we are pushing things to live region outside of render loop.

When pushing things during render loop, the `useLiveRegion()` feels like pure syntactic sugar on `useMemo()` + `useQueueStaticElement()`, than having a true purpose.

What do you think?